netket/logging/runtime_log.py

```diff
@@ -106,17 +106,12 @@ def default(obj):
         return obj.to_dict()
     elif isinstance(obj, np.ndarray):
         if np.issubdtype(obj.dtype, np.complexfloating):
-            return {"real": obj.real, "imag": obj.imag}
-        else:
-            if obj.ndim == 0:
-                return obj.item()
-            elif obj.ndim == 1:
-                return obj.tolist()
-            else:
-                raise TypeError
-
+            return jax.tree_map(
+                np.ascontiguousarray, {"real": obj.real, "imag": obj.imag}
+            )
+    # TODO: replace with isinstance(obj, jnp.ndarray) once all support JAX versions have it
     elif hasattr(obj, "_device"):
```

Comment:
can you try using isinstance and see if it works? The tests with the oldest jax versions should give us a definitive answer about that.

Reply:
Okay, let's find out.

Reply:
seems like its good